src/pb.rs

```diff
@@ -347,18 +351,21 @@ impl<T: Write> ProgressBar<T> {
                 }
             }
         }
+        if !suffix.starts_with(" ") {
+            suffix = format!(" {}", suffix);
+        }
```

Comment:
I would suggest that instead of this, you remove the space from line 334, and then either unconditionally add the space at line 395 or when you create the string at line 326.